InfluxDB.Client.AspNetCore/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,27 @@
+using System.ComponentModel;
+using InfluxDB.Client;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+
+    public static class ServiceCollectionExtensions
+	{
+
+		public static IServiceCollection AddInfluxDBClient(this IServiceCollection serviceCollection, string connectionString, ServiceLifetime serviceLifetime = ServiceLifetime.Scoped)
```

Comment:
Please add tests for `AddInfluxDBClient`. We want to be sure that everything works between releases.